src/index.js

```diff
@@ -0,0 +1,6 @@
+import styles from './assets/scss/app.scss';
```

Comment:
If you don't need access to the styles, just import without assign to variable
```suggestion
import './assets/scss/app.scss';
```
should be enough